packages/components/tabs/src/tab-group.stories.ts

```diff
@@ -1,276 +1,339 @@
+      <sl-tab>Third tab</sl-tab>
+      <sl-tab>Last tab that is longer than the rest</sl-tab>
       <sl-tab-panel>
-        <p>place your router-outlet in this panel</p>
-        <h2 id="output">Active tab:</h2>
+        This example demonstrates how you can have a responsive tab group that adjusts to the width of the page. The
+        scroll container of the tabs has a max width and is centered on the page. There will always be an inline margin,
+        so the tabs never contact the edge of the viewport.
       </sl-tab-panel>
     </sl-tab-group>
   `
 };
+
+export const Selected: Story = {
+  args: {
+    tabs: () => html`
+      <sl-tab>First tab</sl-tab>
+      <sl-tab>Second tab</sl-tab>
+      <sl-tab>Third tab</sl-tab>
+      <sl-tab>Last tab that is longer than the rest</sl-tab>
+    `,
+    tabPanels: () => {
+      const onNext = (event: Event & { target: HTMLElement }) => {
+        const tabGroup = event.target.closest('sl-tab-group') as TabGroup,
+          panel = event.target.closest('sl-tab-panel') as TabPanel,
+          index = Array.from(tabGroup?.querySelectorAll('sl-tab-panel') ?? []).indexOf(panel);
+
+        tabGroup.tabs?.at(index + 1)?.setAttribute('selected', '');
+      };
+
+      const onPrevious = (event: Event & { target: HTMLElement }) => {
+        const tabGroup = event.target.closest('sl-tab-group') as TabGroup,
+          panel = event.target.closest('sl-tab-panel') as TabPanel,
+          index = Array.from(tabGroup?.querySelectorAll('sl-tab-panel') ?? []).indexOf(panel);
+
+        tabGroup.tabs?.at(index - 1)?.setAttribute('selected', '');
+      };
+
+      return html`
+        <sl-tab-panel>
+          <sl-button @click=${onNext}>Focus next tab</sl-button>
+        </sl-tab-panel>
+        <sl-tab-panel>
+          <sl-button @click=${onPrevious}>Focus previous tab</sl-button>
+          <sl-button @click=${onNext}>Focus next tab</sl-button>
+        </sl-tab-panel>
+        <sl-tab-panel>
+          <sl-button @click=${onPrevious}>Focus previous tab</sl-button>
+          <sl-button @click=${onNext}>Focus next tab</sl-button>
+        </sl-tab-panel>
+        <sl-tab-panel>
+          <sl-button @click=${onPrevious}>Focus previous tab</sl-button>
+        </sl-tab-panel>
+      `;
+    }
+  }
+};
+
+export const Sticky: Story = {
+  render: ({ alignTabs }) => html`
+    <style>
+      sl-tab-group::part(container) {
+        position: sticky;
+        inset-block-start: 0;
+      }
+
+      sl-tab-panel {
+        block-size: 200vh;
+      }
+    </style>
+    <sl-tab-group .alignTabs=${alignTabs}>
+      <sl-tab>First tab</sl-tab>
+      <sl-tab>Second tab</sl-tab>
+      <sl-tab disabled>Disabled</sl-tab>
+      <sl-tab>Last tab that is longer than the rest</sl-tab>
+
+      <sl-tab-panel>Contents tab 1</sl-tab-panel>
+      <sl-tab-panel>Contents tab 2</sl-tab-panel>
+      <sl-tab-panel>Contents tab 3</sl-tab-panel>
+      <sl-tab-panel>Contents tab 4</sl-tab-panel>
```

Comment:
The order was tab - panel, tab - panel, tab - panel initially to show that that is a possibility to order the elements in that way. When placing tabs and panels with f.e. Angular it would mean you can have one `for-loop`/`map` to place all your elements, instead of one for the tabs and one for the panels.